methods/config.py

```diff
@@ -1,7 +1,19 @@
-import os
+data_info = {
+        "QNM": {
+            "properties": ["Adhesion", "Deformation", "Dissipation", "LogDMT", "Height", "Stiffness"],
+            "sample_size": {
+                "Backgrounded": 2,
+                "2ComponentFilms": 0.5,
+                "Nanowires": 5
+                }
+            },
+        "AMFM": {
+            "properties": [],
+            "sample_size": 1
+            },
+        "cAFM": {
```

Comment:
Please tell me the order of the properties for this data type. Height first or last

Reply:
Height, then conductivity